app/services/provider_proxy_deployment_v1_service.rb

```diff
@@ -1,6 +1,6 @@
 require_dependency 'apicast/sandbox'
 
-class ProviderProxyDeploymentService
+class ProxyDeploymentV1Service
```

Comment:
then you should do `mv app/services/provider_proxy_deployment_v1_service.rb app/services/proxy_deployment_v1_service.rb`